### ISSUE TITLE
setup "For the Impatient" does not work

### ISSUE THREAD

**Issue by miraculixx** [jrief/djangocms-cascade#130]
I tried this [guide](http://djangocms-cascade.readthedocs.org/en/latest/impatient.html), but it fails scaringly:

```
ImproperlyConfigured at /
Offline compiled file `css/main.css` is missing and libsass has not been installed.
Request Method:	GET
Request URL:	http://localhost:8000/?edit
Django Version:	1.8.8
Exception Type:	ImproperlyConfigured
Exception Value:	
Offline compiled file `css/main.css` is missing and libsass has not been installed.
Exception Location:	/home/demo/projects/cascadedemo/venv/local/lib/python2.7/site-packages/sass_processor/templatetags/sass_tags.py in render, line 90
Python Executable:	/home/demo/projects/cascadedemo/venv/bin/python
Python Version:	2.7.6
Python Path:	
['/home/demo/projects/cascadedemo/venv/src/djangocms-cascade',
 '/home/demo/projects/cascadedemo/venv/src/djangocms-cascade/examples',
 '/home/demo/projects/cascadedemo/venv/src/djangocms-bootstrap3',
 '/home/demo/projects/cascadedemo/venv/src/djangocms-cascade',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/plat-x86_64-linux-gnu',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/lib-tk',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/lib-old',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/lib-dynload',
 '/usr/lib/python2.7',
 '/usr/lib/python2.7/plat-x86_64-linux-gnu',
 '/usr/lib/python2.7/lib-tk',
 '/home/demo/projects/cascadedemo/venv/local/lib/python2.7/site-packages',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/site-packages']
Server time:	Thu, 21 Jan 2016 13:32:08 -0600
Error during template rendering

In template /home/demo/projects/cascadedemo/venv/src/djangocms-cascade/examples/bs3demo/templates/main.html, error at line 5
Offline compiled file `css/main.css` is missing and libsass has not been installed.
```

Installing `pip install libsass` did not help:

```
CompileError at /
Error: File to import not found or unreadable: bootstrap-sass/assets/stylesheets/bootstrap/variables
       Parent style sheet: /home/demo/projects/cascadedemo/venv/src/djangocms-cascade/examples/bs3demo/static/css/_variables.scss
        on line 1 of bs3demo/static/css/_variables.scss
>> @import "bootstrap-sass/assets/stylesheets/bootstrap/variables";
   ^
Request Method:	GET
Request URL:	http://localhost:8000/?edit
Django Version:	1.8.8
Exception Type:	CompileError
Exception Value:	
Error: File to import not found or unreadable: bootstrap-sass/assets/stylesheets/bootstrap/variables
       Parent style sheet: /home/demo/projects/cascadedemo/venv/src/djangocms-cascade/examples/bs3demo/static/css/_variables.scss
        on line 1 of bs3demo/static/css/_variables.scss
>> @import "bootstrap-sass/assets/stylesheets/bootstrap/variables";
   ^
Exception Location:	/home/demo/projects/cascadedemo/venv/local/lib/python2.7/site-packages/sass.py in compile, line 644
Python Executable:	/home/demo/projects/cascadedemo/venv/bin/python
Python Version:	2.7.6
Python Path:	
['/home/demo/projects/cascadedemo/venv/src/djangocms-cascade',
 '/home/demo/projects/cascadedemo/venv/src/djangocms-cascade/examples',
 '/home/demo/projects/cascadedemo/venv/src/djangocms-bootstrap3',
 '/home/demo/projects/cascadedemo/venv/src/djangocms-cascade',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/plat-x86_64-linux-gnu',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/lib-tk',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/lib-old',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/lib-dynload',
 '/usr/lib/python2.7',
 '/usr/lib/python2.7/plat-x86_64-linux-gnu',
 '/usr/lib/python2.7/lib-tk',
 '/home/demo/projects/cascadedemo/venv/local/lib/python2.7/site-packages',
 '/home/demo/projects/cascadedemo/venv/lib/python2.7/site-packages']
Server time:	Thu, 21 Jan 2016 13:48:19 -0600
```

What did I miss?

**Comment by jrief** [comment]
Thanks for reporting. I already added libsass to the dependencies.

For your second problem, please change into folder djangocms-cascade and install

```
npm install bootstrap-sass
```

this should add ``node_modules/bootstrap-sass/...``

then retry plaese.

**Comment by miraculixx** [comment]
thank you, `npm install bootstrap-sass` did the trick. :+1: 

However, there is still a  404 on `http://localhost:8000/static/css/main.css` and the `toggle navigation` button does not seem to effect anything:

![selection_229](https://cloud.githubusercontent.com/assets/859611/12507714/bf16b0d4-c0f7-11e5-8dff-c2d4bc66b62d.jpg)

I'm not trying to be picking, just trying to get it to work following the instructions.

**Comment by jrief** [comment]
For brand new version 0.10.0 I retested the documentation, and in my environment it works.
Please retest, since I am a little biased.

**Comment by rfleschenberg** [comment]
When following the "impatient" docs, after logging in as superuser, I hit a 404 at `http://localhost:8000/en/?edit`. Maybe this is because of my browser's language preference settings.

**Comment by rfleschenberg** [comment]
Other than the small problem fixed by #184, the docs worked for me. Closing the issue.
